Releases.md

```diff
@@ -6,6 +6,24 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.23.0 / 2022.05.27
```

Comment:
I think re-run the version bump script for a patch release

Reply:
Bah... I thought I did that... 🤦